nbagile/docstring.py

```diff
@@ -296,7 +296,6 @@ def clean_file(fname:(Path, str), use_all:bool):
 import nbdev.export2html as exp2html
 from nbdev.imports import read_config_file
 
-@call_parse
 def nbagile_build_lib():
```

Comment:
Bring back these missing `call_parses`, they're needed so they can be exported as a script.

Reply:
Do you know why those would have been dropped?  I didn't do that manually but I can bring them back.  Just thinking if they are autogenerated, I should use the autogenerating process to bring them back

Reply:
Did you generate it with `nbagile`?

If so this is the one case where nbagile cannot build nbagile (weird nits with decorators) IIRC

Reply:
Yeah, I generated everything with nbagile (inside docker-compose)

Reply:
actually I build_docs, but don't build_lib so if I just build_lib with nbdev should that be fine?

Reply:
Yup. You should be able to do `nbagile_build_lib` and be ok

Reply:
Hmm, I tried both `nbagile_build_lib` and `nbdev_build_lib` and neither add the call_parses decorator.  I wonder if I have an old version of something?

Reply:
are the `@call_parse` decorators something you have to manually add to the .py files?

Reply:
nvm, I see them in the ipynb files.  not sure why nbdev wouldn't be bringing those in.  Let me try upgrading

Reply:
I think it must have to do with my version of nbdev.  When I used the version inside my docker-compose it showed up and made quite a few other changes as well